bcftools_1.15.1/bcftools_normvcf_1.15.1.cwl

```diff
@@ -0,0 +1,89 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: bcftools_normvcf
+baseCommand:
+  - bcftools
+  - norm
+inputs:
+  - 'sbg:toolDefaultValue': s
+    id: check-ref
+    type: string?
+    inputBinding:
+      position: 99
+      prefix: '--check-ref'
+  - 'sbg:toolDefaultValue': +
+    id: multiallelics
+    type: string?
+    inputBinding:
+      position: 99
+      prefix: '-m'
+    doc: use any
+    'sbg:fileTypes': fasta
+  - 'sbg:toolDefaultValue': z
+    id: output-type
+    type: string?
+    inputBinding:
+      position: 99
+      prefix: '-O'
+  - id: output-name
```

Comment:
We have specified the outputname, but its not being used in the output that you are globbing